OpenQA/Isotovideo/Runner.pm

```diff
@@ -3,9 +3,14 @@
 
 package OpenQA::Isotovideo::Runner;
 use Mojo::Base -base, -signatures;
+# Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
+# Details: https://progress.opensuse.org/issues/90371
```

Comment:
Please remove the comment here.
You only need to use the module, as I said before.